src/sentry/audit_log/register.py

```diff
@@ -0,0 +1,622 @@
+from sentry.models.auditlogentry import AuditLogEntry
+from sentry.utils.strings import truncatechars
+
+from .manager import AuditLogEvent, AuditLogEventManager
+
+default_manager = AuditLogEventManager()
```

Comment:
Is `AuditLogEventManager` already instantiated in `manager.py`?

Also, to resolve the test failures I think `_event_registry` and `_event_id_lookup` should be instance attributes.